### ISSUE TITLE
Latest stable Docker Hub image uses expired GPG key

### ISSUE THREAD

**Issue by wfriesen** [jellyfin/jellyfin#4830]
**Describe the bug**
I use the `jellyfin/jellyfin` image as a base and install some other software within it (namely, comskip and it's dependencies). When rebuilding the image now I get this error:

```
W: GPG error: https://repo.jellyfin.org/debian buster InRelease: The following signatures were invalid: EXPKEYSIG 49023CD01DE21A7B Jellyfin Team <team@jellyfin.org>
E: The repository 'https://repo.jellyfin.org/debian buster InRelease' is not signed.
```

The steps outlined in https://jellyfin.org/posts/jellyfin-apt-key/ are not applicable, as the image does not include the packages required to manipulate keys, and apt is broken.

**System (please complete the following information):**
 - OS: Ubuntu
 - Virtualization: Docker
 - Jellyfin Version: Docker Hub: jellyfin/jellyfin:latest
 
**To Reproduce**
Attempt to build an image from this Dockerfile:

```Dockerfile
FROM jellyfin/jellyfin

RUN apt-get update
```

**Expected behavior**
Successful image created.

**Logs**
```
W: GPG error: https://repo.jellyfin.org/debian buster InRelease: The following signatures were invalid: EXPKEYSIG 49023CD01DE21A7B Jellyfin Team <team@jellyfin.org>
E: The repository 'https://repo.jellyfin.org/debian buster InRelease' is not signed.
```

**Comment by Artiume** [comment]
This will be fixed upon 10.7 release

**Comment by nagy511** [comment]
`FROM jellyfin/jellyfin:latest
RUN rm /etc/apt/sources.list.d/jellyfin.list

RUN apt update && apt upgrade -y

RUN  apt install apt-transport-https gnupg wget -y

RUN wget -O - https://repo.jellyfin.org/jellyfin_team.gpg.key | apt-key add -
RUN echo "deb [arch=$( dpkg --print-architecture )] https://repo.jellyfin.org/$( awk -F'=' '/^ID=/{ print $NF }' /etc/os-release ) $( awk -F'=' '/^VERSION_CODENAME=/{ print $NF }' /etc/os-release ) main" | tee /etc/apt/sources.list.d/jellyfin.list`

This works for me until the 10.7 release ;)
